backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -41,4 +42,9 @@ include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
 
+[id="troubleshooting-mcg"]
+== Troubleshooting MCG
+
+include::modules/oadp-mcg-performance-tuning.adoc[leveloffset=+2]
```

Comment:
Sorry, feels like some kind of context is missing. It's not clear why performance tuning related module is added under a troubleshooting heading.